webapp/pages/post/_id/_slug/index.vue

```diff
@@ -179,6 +183,9 @@ export default {
       if (!author) return false
       return this.$store.getters['auth/user'].id === author.id
     },
+    sortedTags() {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-03T14:51:33Z" title="Wednesday, June 3rd 2020, 4:51:33 pm +02:00">Jun 3, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -179,6 +179,20 @@ export default {
       if (!author) return false
       return this.$store.getters['auth/user'].id === author.id
     },
+    sortedTags() {
```


I would love to have this imported as a more general function located in `components/utils/PostHelpers` or even in a more general utility file.
This way it can be reused in `webapp/pages/post/_id/_slug/more-info.vue`. 👇🏼

It can be used to sort the post categories in the future. 🙏🏼
A missing issue as far as I see …